staticaddr/withdraw/manager.go

```diff
@@ -577,7 +580,7 @@ func (m *Manager) publishFinalizedWithdrawalTx(ctx context.Context,
 			return false, nil
 		}
 	} else {
-		log.Debugf("published deposit withdrawal with txid: %v",
+		log.Debugf("Published deposit withdrawal with txid: %v!",
```

Comment:
nit: when copying the txid from the log the `...` and `!` might be copied too, not sure if we want that.